### PR TITLE
Use verbosity

### PR DESCRIPTION
**Pull request by polluks**
How to PR two files at once using GitHub's front-end?

### CODE REVIEW

File: src/ar65/list.c [hunk 1]
```diff
@@ -73,6 +76,10 @@ void ListObjFiles (int argc, char* argv [])
         /* Get the entry */
         O = CollConstAt (&ObjPool, I);
 
+        /* Print the size */
+        if (Verbosity > 0) {
+            printf ("%5ld ", O->Size);
+        }
```
Comment (greg-king5) [inline src/ar65/list.c]: Shouldn't you use `Print()` for that?
```C
Print (stdout, 1, "%5lu ", O->Size);
```

**Comment by oliverschmidt** [comment]
> How to PR two files at once using GitHub's front-end?

Maybe https://github.com/WGBH/pbucore/wiki/Contributing-to-the-project-through-Github-web-interface works for you.

File: src/ar65/list.c (hunk 1 above)
Comment (polluks) [inline src/ar65/list.c]: indeed

File: src/ar65/list.c [hunk 2]
```diff
@@ -78,7 +78,7 @@ void ListObjFiles (int argc, char* argv [])
 
         /* Print the size */
         if (Verbosity > 0) {
```
Comment (greg-king5) [inline src/ar65/list.c]: That test of `Verbosity` is built into `Print()`; it shouldn't be done here.

Comment (polluks) [inline src/ar65/list.c]: Ah, now I got it!

**Comment by polluks** [comment]
Well, I found no hint for multiple files.
"Submitting a change to this file will write it to a new branch", there's nothing to choose.

**Comment by greg-king5** [comment]
> Well, I found no hint for multiple files.
> "Submitting a change to this file will write it to a new branch"; there's nothing to choose.

That's strange!  When I try it, I get a web page that looks like this:

image

Near the bottom of that page are two "radio buttons".  The first one says, "Commit directly to the `<so-and-so>` branch".  The second one says, "Create a new branch for this commit and start a pull request."

That first one (the default) lets you combine several files into a single PR.  You must do each step separately:

1. Go to the master branch; then, create a new branch (do that on your fork's home page).
2. Navigate to each file that you want to change.  Edit it.  Commit it directly to the current branch.  Repeat with another file.
3. When you have finished editting and committing all of the files, go back to the home page.  You will see a box near the top that shows "Your recently pushed branches".  Click the Pull Request button for the branch that you just finished editting.

When you want to add a file to an open PR, choose that PR's branch, instead of doing step number 1.  And, don't do step number 3.

**Comment by oliverschmidt** [comment]
Hi,

I'm in general happy about every contribution to cc65 and I certainly don't want to be a spoilsport but by now we have (as far as i understand) three pull requests for this topic. And every pull request has its own discussion. Pretty please don't give me such a hard time to follow this !!!

**Comment by polluks** [comment]
Oh boy, I opened Pandora's box!

**Comment by oliverschmidt** [comment]
Do I understand it correctly that https://github.com/cc65/cc65/pull/442, https://github.com/cc65/cc65/pull/443 and https://github.com/cc65/cc65/pull/444 are ready to be merged?

**Comment by polluks** [comment]
Anyway next time I will try Greg's steps.

**Comment by oliverschmidt** [comment]
> Anyway next time I will try Greg's steps.

:-)

**Comment by polluks** [comment]
@greg-king5 sorry, I don't see "Your recently pushed branches" of your third point. I can only create a PR at my branch which doesn't make any sense.

**Comment by greg-king5** [comment]
> Sorry, I don't see "Your recently pushed branches" of your third point. I can only create a PR at my branch -- which doesn't make any sense.

It should appear just under a fat bar that extends across the page; that bar appears under some statistics about the repository: number of commits, number of branches, number of releases, and number of contributors.

Also, Github really is serious about that "recently" adverb.  If you wait too long between your last commit and going to the home page, then there won't be any **_recently_** pushed stuff.

If you use your browser's "back" button to get to the home page, then refresh the page.

**Comment by greg-king5** [comment]
I forgot to write this:
Even if you don't see that "recently pushed" message, there are two other "pull request" buttons that seem to do the job (at least, for me):

- "`New pull request`" (next to the name of the current branch)
- "`Pull request`" (next to a "`Compare`" button, and in a box that says how many commits behind/even/ahead of the upstream master branch)

**Comment by polluks** [comment]
Hmm, maybe I cannot see it because my branch is out of sync. If I PR file by file it doesn't matter. I'm just guessing.

**Pull request merged**